packages/indexer-common/src/indexer-management/__tests__/helpers.ts

```diff
@@ -243,27 +243,27 @@ describe('Actions', () => {
   })
 })
 describe('Types', () => {
-  test('Fail to resolve chain id', async () => {
-    await expect(resolveChainId('arbitrum')).rejects.toThrow(
-      'Failed to resolve CAIP2 ID from the provided network alias: arbitrum',
+  test('Fail to resolve chain id', () => {
+    expect(() => resolveChainId('arbitrum')).toThrow(
+      /^Failed to resolve CAIP2 ID from the provided network alias: arbitrum$/,
```

Comment:
Could you explain the change here from string (wrapped in single quotes) to the new string without quotes and regex looking prefix/suffix?

Reply:
Fixed in a0848612